exp-1-merit/kafka-shared/modules/producer/variables.tf

```diff
@@ -0,0 +1,84 @@
+# ACL
+variable "resource_name" {
+  type        = string
+  description = "The name of the Kafka resource for which the ACL is being created."
+}
+
+variable "resource_type" {
+  type        = string
+  description = "The type of the Kafka resource for which the ACL is being created. Valid values: 'Unknown', 'Any', 'Topic', 'Group', 'Cluster', or 'TransactionalID'."
+  default     = "Topic"
+
+  validation {
+    condition     = contains(["Unknown", "Any", "Topic", "Group", "Cluster", "TransactionalID"], var.resource_type)
+    error_message = "Invalid resource_type. Must be one of: 'Unknown', 'Any', 'Topic', 'Group', 'Cluster', or 'TransactionalID'."
+  }
+}
+
+variable "policy_user" {
```

Comment:
This should be the same as https://github.com/utilitywarehouse/kafka-cluster-config/pull/27/files#r1275074869